### PR TITLE
use function for generated default messages

### PR DESCRIPTION
**Pull request by akozenko**

### CODE REVIEW

File: src/directive.js [hunk 1]
```diff
@@ -17,8 +18,11 @@
                  * @returns {}
                  */
                 var validFunc = function(element, validMessage, validation, callback, ctrl) {
-                    var messageElem = element.next(),
-                        messageToShow = validMessage || $validationProvider.getDefaultMsg(validation).success;
+                    var messageElem, node = document.querySelector('[validator-msg-for="' + ctrl.$name + '"]');
```
Comment (huei90) [inline src/directive.js]: @akozenko May I know why there's the attribute `validator-msg-for=""` ?

Comment (akozenko) [inline src/directive.js]: Hi.
In december i don't seen request #70
<https://github.com/huei90/angular-validation/pull/70>
but I need change location for some messages, so I use this attribute for
resolve my problems.

2015-02-11 15:42 GMT+02:00 Huei Tan <notifications@github.com>:

Comment (huei90) [inline src/directive.js]: Sorry for the late update.

We just update it by using `message-id`

**Comment by huei90** [comment]
Reopen it if having other issue.

**Pull request closed without merging**